components/LessonTitleCard.tsx

```diff
@@ -44,7 +44,8 @@ const LessonTitleCard: React.FC<LessonTitleProps> = ({
                   // Link does not get correct styles without the href
                   // Prevent scroll to top before going back
                   e.preventDefault()
-                  router.back()
+
+                  router.push('/')
```

Comment:
Do we always want to redirect to homepage when a user clicks `Go back`?

Let's say I'm looking at js1 challenges. And then I click the `lesson` button that shows the lesson document for js1. Currently, if I click `Go back`, then I will land back on the js1 challenges page. 

What I believe your change will do is it will always redirect users back to the homepage -- is that what we want? IMO I think its a better user experience to redirect users to the last page they were on, and only redirect to the homepage if the redirectUrl in question is not c0d3.com domain

Reply:
I'll look into it

Reply:
My first thought is it needs a global state that will be true if the user accessed any page in c0d3.
```suggestion
                inC0d3 ? router.back() : router.push('/')
```